src/entry_node.cpp

```diff
@@ -77,107 +79,20 @@ Status EntryNode::fetchResults(BlobMap& outputs) {
                 SPDLOG_LOGGER_DEBUG(dag_executor_logger, "[Node: {}] Missing input with specific name: {}", getName(), details);
```

Comment:
Do we validate missing inputs twice? Here and inside `deserializePredictRequest`? Do we even need this loop after unification? The log inside this loop is misleading since it is not really the moment of deserializing. Deserialization happens afterwards, inside `deserializePredictRequest`

Reply:
Leftover debug from test fixing.